app/config/factionhall/events.json

```diff
@@ -1,52 +1,60 @@
 {
     "events": [
         {
-            "name": "Ace Presentation by TST Florida",
-            "description": "A collaborative project from several ace members of the Florida congregation compiled for Lupercalia.",
-            "room": "rotunda",
-            "start": "2021-06-26T20:00:00-04:00",
-            "end": "2021-06-26T20:30:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/pride/rainbowbaph.png"
+            "name": "The Politics of Moral Panic: How Polarization Undermined Collective Reason by Lucien Greaves",
+            "description": "Join Lucien Greaves to discuss The Politics of Moral Panic: How Polarization Undermines Collective Reason.",
+            "room": "auditorium",
+            "start": "2022-08-20T20:00:00-04:00",
+            "end": "2022-08-20T21:00:00-04:00",
+            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/calendarhall.png"
         },
         {
-            "name": "Sex Work and Cupcakes: Conversations about identity with Ada King, Mischief Madness & Chalice Blythe",
-            "description": "Join Chalice Blythe, Mischief Madness and Ada King for unwrapping presents from @divinetoys, cupcakes and conversations about not being straight while doing sex work.",
-            "room": "auditorium",
-            "start": "2021-06-26T20:30:00-04:00",
-            "end": "2021-06-26T21:30:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/pride/rainbowbaph.png"
+            "name": "The Rise and Fall of Bennett Braun by Evan Anderson",
+            "description": "Join Grey Faction director Evan Anderson to discuss The Rise and Fall of Bennett Braun.",
+            "room": "alice-tallmadge",
```

Comment:
Please revert line 14 to the previous room ID. I believe it was "hall". Confirm what it was please before doing this. Thanks!